.github/workflows/call.setup-deploy-and-test-local-subgraph.yml

```diff
@@ -0,0 +1,82 @@
+name: Set up, deploy and test local subgraph
```

Comment:
Maybe we should be consistent about applying https://erinwrightwriting.com/what-is-headline-style-capitalization/ to name fields...